src/navigation/navigation.tsx

```diff
@@ -415,3 +425,48 @@ const NotificationsStack = (): JSX.Element => {
     </Stack.Navigator>
   )
 }
+
+const OnboardingStack = (): JSX.Element => {
+  return (
+    <Stack.Navigator
+      initialRouteName={ScreenRoutesEnum.ONBOARDING_WELCOME}
+      screenOptions={{
+        animation: 'none'
+      }}
+    >
+      <Stack.Screen
+        name={ScreenRoutesEnum.ONBOARDING_WELCOME}
+        component={SSIOnboardingWelcomeScreen}
+        options={{
+          headerShown: false
+        }}
+      />
+    </Stack.Navigator>
+  )
+}
+
+/**
+ * WIP. Figuring out how to properly navigate between certain flows and making sure that the user cannot navigate back to the previous ones
```

Comment:
WIP still needed?

Reply:
ye, i would like to keep it for now to make clear that this is not done and might not be the final approach